drivers/src/nrfx_spim.c

```diff
@@ -590,7 +602,7 @@ void nrfx_spim_abort(nrfx_spim_t const * p_instance)
     NRFX_ASSERT(p_cb->state != NRFX_DRV_STATE_UNINITIALIZED);
 
     nrf_spim_task_trigger(p_instance->p_reg, NRF_SPIM_TASK_STOP);
-    while (!nrf_spim_event_check(p_instance->p_reg, NRF_SPIM_EVENT_STOPPED))
+    while (p_cb->transfer_in_progress && !nrf_spim_event_check(p_instance->p_reg, NRF_SPIM_EVENT_STOPPED))
```

Comment:
Should we add comment for the change of this line?

Reply:
add comments to explain why.